lib/utils.js

```diff
@@ -47,11 +39,28 @@ const defaults = {
   messageOnly: false,
   tz: 'UTC',
   newline: false,
-  sync: false
+  sync: false,
+  pinoLevelToSyslogSeverity
+}
+
+function buildOptions (options = {}, args = {}) {
+  const opts = Object.assign({}, defaults, options, args)
+
+  if (options.customLevels == null) {
+    return opts
+  }
+
+  for (const [key, { level, syslogSeverity }] of Object.entries(options.customLevels)) {
+    pinoLevels[key] = level
+    pinoLevelToSyslogSeverity.set(pinoLevels[key], severity[syslogSeverity])
```

Comment:
This is problematic because it changes a global variable. Refactor your code to avoid it.